### PR TITLE
[pango] Add missing link library

### PR DESCRIPTION
**Pull request by janisozaur**
Gdi32 is used by pango, but not linked to. This works on x86, but not on
ARM.

### CODE REVIEW

**Comment by ras0219-msft** [comment]
Thanks for the PR!

The CI shows no changes (this didn't fix any platforms). What specific build should this change fix?

File: ports/pango/CMakeLists.txt [hunk 1]
```diff
@@ -118,7 +118,7 @@ pango_add_module(pangowin32
     pango/pangowin32-fontcache.c
     pango/pangowin32-fontmap.c
     pango/pangowin32-shape.c)
-target_link_libraries(pangowin32 usp10 pango)
+target_link_libraries(pangowin32 usp10 pango gdi32)
```
Comment (janisozaur) [inline ports/pango/CMakeLists.txt]: This is the change

**Review by janisozaur — COMMENTED** [review]
I added gdi32 library that is required to link pangowin32 project correctly

**Comment by janisozaur** [comment]
It was failing in `arm64-windows` triplet before this change

**Review by c72578 — COMMENTED** [review]
The line endings in **`ports/pango/portfile.cmake`** have been changed from Windows (CR LF) to Unix (LF). Any special reason for this?

**Comment by janisozaur** [comment]
Yes: they were crlf.

**Comment by ras0219-msft** [comment]
Currently, our CI system isn't able to build `pango` for `arm64-windows` because it transitively requires `libffi` (via `glib`) which fails to build for `arm64-window`. How are you able to build `pango:arm64-windows`?

**Comment by janisozaur** [comment]
I have used https://github.com/microsoft/vcpkg/pull/6119, but libffi upstream has been ported to support msvc arm64 already.

**Comment by ras0219-msft** [comment]
LGTM, thanks!

**Pull request merged**